assets/js/components/Button.js

```diff
@@ -118,7 +118,16 @@ const Button = forwardRef(
 						tooltip: 'googlesitekit-tooltip',
 					} }
 				>
-					{ ButtonComponent }
+					{
+						// If the button is disabled it has no events for the Tooltip
+						// component to use, so it needs to be wrapped in a containing
+						// element.
+						disabled ? (
+							<span>{ ButtonComponent }</span>
+						) : (
+							ButtonComponent
+						)
```

Comment:
I'm not sure if this is the right solution here. Would you please revert this change for now and we can address it separately?

Reply:
Sure 👍
Just for context, this is where the solution came from: https://mui.com/components/tooltips/#disabled-elements
Happy to revisit as a separate issue.